### PR TITLE
Add bpfd to the emerging applications

### PR DESCRIPTION
**Pull request by astoycos**
Add bpfd(https://github.com/bpfd-dev/bpfd) to the emerging application list.

- [x] The project must be open source. All source code must be licensed under an open source license. Any documentation must be licensed under an open license. -> https://github.com/bpfd-dev/bpfd#license

- [x] The project must be using eBPF as its underlying core technology (in other words, a project would lose its purpose if the eBPF parts are removed) or help accelerate the adoption of eBPF in production. -> https://github.com/bpfd-dev/bpfd#what-is-bpfd

- [x] The project must be open to collaboration and have a governance model following open source best-practices. -> https://github.com/bpfd-dev/bpfd/blob/main/GOVERNANCE.md

### CODE REVIEW

**Comment by xmulligan** [comment]
Thanks for the addition! Looks like the build didn't work. I would guess it might have to do with the ` around the description. Can you try again?

File: src/components/pages/project-landscape/project-card/project-card.jsx [hunk 1]
```diff
@@ -32,6 +32,7 @@ import sysinternalsLogo from './logos/sysinternals.svg';
 import traceeLogo from './logos/tracee.svg';
 import sshlogLogo from './logos/sshlog.svg';
 import carettaLogo from './logos/caretta.svg';
+import bpfdLogo from `./logos/bpfd.svg`;
```
[the remark this answers is not in the document]
Comment (astoycos) [reply src/components/pages/project-landscape/project-card/project-card.jsx:35]: Good catch!! Thanks @Billy99

File: src/data/applications/emerging.js [hunk 2]
```diff
@@ -335,6 +335,21 @@ const emergingApplications = [
       { label: 'GitHub', url: 'https://github.com/sshlog/agent/' },
       { label: 'Website', url: 'http://www.sshlog.com/' },
     ],
+  },
+  {
+    logoUrl: 'https://github.com/bpfd-dev/bpfd',
+    name: 'bpfd',
+    logoName: 'bpfdLogo',
+    title: 'A system daemon for managing eBPF programs',
+    description: `bpfd is a system daemon aimed at simplifying the deployment and
```
Comment (Billy99) [inline src/data/applications/emerging.js:344]: Beginning and End
```suggestion
    description: 'bpfd is a system daemon aimed at simplifying the deployment and
```

**Comment by astoycos** [comment]
The previous netlify build passed @xmulligan I just pushed a tiny change for the title, specifically changing it to 

-> `A system daemon and Kubernetes operator for managing eBPF programs`

And it should be all set now, thanks!!

**Comment by xmulligan** [comment]
Thanks for the addition! Looking forward to seeing how the project grows

**Pull request merged**